docs/src/tutorial/index.rst

```diff
@@ -0,0 +1,16 @@
+********
+Tutorial
+********
+
+These tutorials will guide you to create a :term:`Nextstrain build <docs.nextstrain.org:build>` for genomic surveillance.
```

Comment:
I just realized that we don't provide any preface for readers to explain the overall goals of the tutorial, a list of [learning outcomes](https://teaching.cornell.edu/teaching-resources/designing-your-course/setting-learning-outcomes), or the scope of the tutorial (e.g., we're not going to cover everything in genomic epidemiology). The main index page provides the overall goals with:

> This template and tutorial will walk you through the process of running a basic phylogenetic analysis on SARS-CoV-2 data. We’ve created these resources with the goal of enabling Departments of Public Health to start using Nextstrain to understand their SARS-CoV-2 genomic data within 1-2 hours.

This text could live in an "Introduction" page of the tutorial (instead of the top-level docs where it feels out of place now). Then we could expand on these goals to include learning outcomes and details about the scope like:

```rst
At the end of this tutorial, you will be able to:

- create phylogenetic trees of SARS-CoV-2 genomes from different sources including GISAID and Nextstrain-curated GenBank data
- visualize the resulting trees in Auspice
- define subsampling logic for your own genomic epidemiological analysis
- (optional?) configure parameters of the SARS-CoV-2 workflow that produces the phylogenetic trees
- (optional?) configure how your phylogenetic trees and associated metadata appear in Auspice
 
After completing this tutorial, you may wish to `learn more about genomic epidemiology <link to Alli's gen epi book or some other resources?>`_ or `review all possible options to configure your SARS-CoV-2 analyses with Nextstrain <../reference/configuration.html>`_.
```

Reply:
Done: 534c9c05